public/app/features/plugins/sql/components/QueryEditor.tsx

```diff
@@ -13,7 +13,7 @@ import { QueryHeader } from './QueryHeader';
 import { RawEditor } from './query-editor-raw/RawEditor';
 import { VisualEditor } from './visual-query-builder/VisualEditor';
 
-type Props = QueryEditorProps<SqlDatasource, SQLQuery, SQLOptions>;
+export type Props = QueryEditorProps<SqlDatasource, SQLQuery, SQLOptions>;
```

Comment:
Super nit picking here. I think we don't need to export this.
```suggestion
type Props = QueryEditorProps<SqlDatasource, SQLQuery, SQLOptions>;
```